microsetta_public_api/resources.py

```diff
@@ -12,7 +12,7 @@
 from q2_types.feature_data import FeatureData, Taxonomy
 from q2_types.ordination import PCoAResults
 
-from microsetta_public_api.logging import timeit
+from microsetta_public_api._logging import timeit
```

Comment:
Is this change necessary/in-scope?

Reply:
Kept encountering (https://stackoverflow.com/questions/4482587/module-object-has-no-attribute-basicconfig). Having a module named `logging.py` can create conflicts with the `logging.py` from site-packages

Reply:
Alright sounds good.